common/src/objects/workload_state.rs

```diff
@@ -291,6 +344,20 @@ impl ExecutionState {
             ..Default::default()
         }
     }
+
+    pub fn waiting_to_start() -> Self {
+        ExecutionState {
+            state: ExecutionStateEnum::Pending(PendingSubstate::WaitingToStart),
+            additional_info: "waiting for workload dependencies.".to_string(),
+        }
+    }
+
+    pub fn waiting_to_stop() -> Self {
+        ExecutionState {
+            state: ExecutionStateEnum::Stopping(StoppingSubstate::WaitingToStop),
+            additional_info: "waiting for workload dependencies.".to_string(),
```

Comment:
I think it would be better if the string shows the names of the workloads it waits for (for waiting to stop and for waiting to start).
And the Ank CLI can limit the amount of characters if a workload has too much dependency names.

What do you think?

Reply:
This would be nice. But if this addition info is not updated for already running dependencies, this could be misleading and updating this additional info would be to much work. Just remove the additional_info.